Documentation/HowToGuides/AddingAnInteractor/README.md

```diff
@@ -0,0 +1,68 @@
+
+## Prerequisites
+
+* [Install the Tilia.CameraRigs.TrackedAlias.Unity] package dependency in to your [Unity] project.
+* [Install the Tilia.Interactions.Interactables.Unity] package dependency in to your [Unity] project.
+
+## Let's Start
+
+### Step 1
+
+The `Interactions.Interactor` prefab comes with an `ExampleAvatar` GameObject, which provides a basic representation of the Interactor containing a simple cube mesh and a simple cube collider. These can be changed to suit whatever purpose you may have, but for this example, we’ll keep the `ExampleAvatar` GameObject untouched.
+
+> If you followed the [Adding A TrackedAlias] guide and added in custom GameObjects for controller avatars, you can delete them now as the `ExampleAvatar` GameObject will provide us with a basic controller avatar instead.
+
+Expand the `Tilia Interactors Interactables Unity` Package directory in the Unity Project window and then select -> `Packages -> Tilia.Interactors.Interactables.Unity -> Runtime -> Interactors -> Prefabs`. For this example, we’re going to add the `Interactions.Interactor` prefab to our controller aliases within our `TrackedAlias` prefab, so expand the `TrackedAlias` prefab in the Unity Hierarchy window until the `LeftControllerAlias` and `RightControllerAlias` GameObjects are visible.
+
+Drag and drop the `Interactions.Interactor` prefab to be a child of the `LeftControllerAlias` then drag and drop another `Interactions.Interactor` prefab to be a child of the `RightControllerAlias`.
+
+![Making Interactor Prefab AChild Of Left And Right Alias](assets/images/MakingInteractorPrefabAChildOfLeftAndRightAlias.png)
+
+Note: The following steps will be carried out on both the `LeftControllerAlias` and the `RightControllerAlias` but the instructions will only outline the steps once, so be sure to carry them out for both controller aliases.
+
+### Step 2
+
+Select the `Interactions.Interactor` prefab in the Unity Hierarchy under the `LeftControllerAlias` GameObject and change the `Interactor Facade` component to configure the base functionality of the Interactor.
+
+We need to specify an action that initiates the Interactor’s Grab notification. The Grab Action parameter on the Interactor Facade allows us to provide a Boolean Action to initiate the Grab function of the Interactor.
+
+For this example we will set up the pointer to use the simulator controls utilizing the mouse.
+
+Expand `CameraRigs.SpatialSimulator -> Input -> Control Objects Input -> Right Controller Input -> Buttons`.
+
+`Button One` is the left mouse button and we'll use this to activate the grab action on our left interactor so drag and drop the `Button One` GameObject into the `Grab Action` parameter on the Interactor Facade component on the `Interactions.Interactor` prefab nested inside `LeftControllerAlias` Gameobject
+
+![Drag And Drop Button One Into Grab Activation Parameter](assets/images/DragAndDropButtonOneIntoGrabActivationParameter.png)
```

Comment:
The graphic is confusing. The first graphic shows the Interactor prefabs dragged into the tracked alias, but then this graphic shows the interactor not in the tracked alias.

Reply:
Needs a note field underneath here to say do the same with `Button Two` on the right alias, like:

```
> Note: Be sure to do the same with the `Button Two` GameObject and the `RightControllerAlias -> Interactions.Interactor` prefab.
```